### ISSUE TITLE
building os from your instruction getting error

### ISSUE THREAD

**Issue by phantu91git** [Yura80/pikvm-rock64#9]
i am building os from your instruction but getting the repo authentication error
is there a way to build the os locally without repo url?
is there a way to upgrade the kvmd and the web interface to the lastest for orange pi zero?
i am quite new so plz help

**Comment by Gizmo-Ger** [comment]
I´m also having a hard time to build as well. Unfortunalty this fork seems pretty much not maintained anymore since the last commit was in December 2020. I also would like to know how to build locally and adjust settings for an Odroid C2 I´m having laying around here.

**Comment by phantu91git** [comment]
> 
> 
> I´m also having a hard time to build as well. Unfortunalty this fork seems pretty much not maintained anymore since the last commit was in December 2020. I also would like to know how to build locally and adjust settings for an Odroid C2 I´m having laying around here.

ya it is hard to follow the instruction, it show error with the repo authentication of the author.
the repo of pikvm have kvmd over 3x and webterr and full-screen size option on broweser, we need someway to work arround for that

**Comment by danfmihai** [comment]
Same here. Same error:
```
Sending build context to Docker daemon  15.36kB
Step 1/5 : FROM archlinux/base
pull access denied for archlinux/base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
make[1]: *** [Makefile:197: toolbox] Error 1
make[1]: Leaving directory 'projects/orangepi/packages/.pi-builder/generic-aarch64'
```
Followed the instructions....

**Comment by gigobait** [comment]
Dan, that's a simple on lol. Ive been hammering away at this for weeks on and off. Your error just means you need to change archlinux/base to archlinux:base or archlinux:latest (i believe). Both will work as long as the syntax is correct. Just serach for ALL files with the words "archlinux/base" and you should only find 1.

**Comment by gigobait** [comment]
> Same here. Same error:
> 
> ```
> Sending build context to Docker daemon  15.36kB
> Step 1/5 : FROM archlinux/base
> pull access denied for archlinux/base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
> make[1]: *** [Makefile:197: toolbox] Error 1
> make[1]: Leaving directory 'projects/orangepi/packages/.pi-builder/generic-aarch64'
> ```
> Dan look at my directions
> Followed the instructions....
